user/user.go

```
@@ -0,0 +1,55 @@
+package user
+
+import (
+	"encoding/json"
+	"net/http"
+	"timezone-converter/db"
+
+	"github.com/gorilla/mux"
+)
+
+type User struct {
+	Id        string                    `json:"id"`
+	Username  string                    `json:"username"`
+	Password  string                    `json:"password"`
+	Timeslots map[string]TimeslotStatus `json:"timeslots"`
+}
+
+type TimeslotStatus struct {
+	Booked bool `json:"booked"`
+}
+
+func Register(w http.ResponseWriter, r *http.Request) {
+	// get posted data for user and store into db
+	var u User
+	defaultTimeslots := map[string]TimeslotStatus{
+		"9:00":  {Booked: false},
+		"10:00": {Booked: false},
+		"11:00": {Booked: false},
+		"12:00": {Booked: false},
+		"13:00": {Booked: false},
+		"14:00": {Booked: false},
+		"15:00": {Booked: false},
+		"16:00": {Booked: false},
+	}
+
+	u.Timeslots = defaultTimeslots
+	jsonWithTimeslots, _ := json.Marshal(defaultTimeslots)
```

Do not suppress errors, handle them.